skills/plan-ceo/SKILL.md

```
@@ -0,0 +1,327 @@
+- EXISTING SPECS: overlapping specs and their status
+- PAIN POINTS: known issues from TODO.md relevant to the plan
+- RETROSPECTIVE: prior review cycle findings (if any)
+- TASTE CALIBRATION: style references and anti-patterns
+- EXISTING CODE: code that already partially solves problems in this plan"
+description: "System audit"
+run_in_background: true
+```
+
+**Do NOT wait for this agent.** Proceed immediately to Step 0. When the agent completes, incorporate its findings into the review. If the agent hasn't returned by Section 1, check for its results then.
+
+## Step 0: Nuclear Scope Challenge + Mode Selection
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)